### PR TITLE
mat64: add missing doc strings

### PR DESCRIPTION
**Pull request by kortschak**
@btracey

### CODE REVIEW

File: mat64/index_bound_checks.go [hunk 1]
```diff
@@ -24,6 +25,7 @@ func (m *Dense) at(r, c int) float64 {
 	return m.mat.Data[r*m.mat.Stride+c]
 }
 
+// SetSym sets the element at row r, column c to the value v. It panics if c is not zero.
```
Comment (btracey) [inline mat64/index_bound_checks.go]: This is a nice frankencopy message :). Function name from SymDense, behavior of Vector, and attached to Dense. I like it.

File: mat64/index_bound_checks.go [hunk 2]
```diff
@@ -52,6 +55,7 @@ func (m *Vector) at(r int) float64 {
 	return m.mat.Data[r*m.mat.Inc]
 }
 
+// SetSym sets the element at (r,c) to the value v. It panics if c is not zero.
```
Comment (btracey) [inline mat64/index_bound_checks.go]: wrong name

File: mat64/index_bound_checks.go [hunk 3]
```diff
@@ -126,7 +130,7 @@ func (t *TriDense) at(r, c int) float64 {
 	return t.mat.Data[r*t.mat.Stride+c]
 }
 
-// SetTri sets the element of the triangular matrix at row r and column c.
+// SetTri sets the element of the triangular matrix at row r, column c.
```
Comment (btracey) [inline mat64/index_bound_checks.go]: to the value v.

File: mat64/index_no_bound_checks.go [hunk 4]
```diff
@@ -24,6 +25,7 @@ func (m *Dense) at(r, c int) float64 {
 	return m.mat.Data[r*m.mat.Stride+c]
 }
 
+// SetSym sets the element at row r, column c to the value v. It panics if c is not zero.
```
Comment (btracey) [inline mat64/index_no_bound_checks.go]: see above.

File: mat64/index_no_bound_checks.go [hunk 5]
```diff
@@ -52,6 +55,7 @@ func (m *Vector) at(r int) float64 {
 	return m.mat.Data[r*m.mat.Inc]
 }
 
+// SetSym sets the element at (r,c) to the value v. It panics if c is not zero.
```
Comment (btracey) [inline mat64/index_no_bound_checks.go]: Set

File: mat64/index_bound_checks.go (hunk 1 above)
Comment (kortschak) [inline mat64/index_bound_checks.go]: William S. Burrows is a significant influence in my work.

**Comment by kortschak** [comment]
Ugh. PTAL

**Comment by btracey** [comment]
LGTM

**Pull request merged**